django/db/models/sql/query.py

```diff
@@ -1085,12 +1080,10 @@ def check_alias(self, alias):
                 "semicolons, or SQL comments."
             )
 
-    def add_annotation(self, annotation, alias, is_summary=False, select=True):
+    def add_annotation(self, annotation, alias, select=True):
```

Comment:
If this is deemed problematic it could go through a deprecation path; it just happens to be unused by the core now.

Reply:
I added a small release note.